subsys/net/lib/nrf_cloud/src/nrf_cloud_alerts.c

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <net/nrf_cloud.h>
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+#include <date_time.h>
+#include "nrf_cloud_fsm.h"
+#include "nrf_cloud_mem.h"
+#include "nrf_cloud_codec.h"
+#include "nrf_cloud_transport.h"
+#include <net/nrf_cloud_rest.h>
+#include <net/nrf_cloud_alerts.h>
+
+LOG_MODULE_REGISTER(nrf_cloud_alerts, CONFIG_NRF_CLOUD_LOG_LEVEL);
```

Comment:
does it need its own log ctrl?
```suggestion
LOG_MODULE_REGISTER(nrf_cloud_alerts, CONFIG_NRF_CLOUD_ALERTS_LOG_LEVEL);
```